src/Core/src/PublicAPI/net-tizen/PublicAPI.Shipped.txt

```diff
@@ -312,7 +312,6 @@ Microsoft.Maui.FontManager
 Microsoft.Maui.FontManager.DefaultFontSize.get -> double
 Microsoft.Maui.FontManager.FontManager(Microsoft.Maui.IFontRegistrar! fontRegistrar, System.IServiceProvider? serviceProvider = null) -> void
 Microsoft.Maui.FontManager.GetFont(Microsoft.Maui.Font font) -> string!
-Microsoft.Maui.FontManager.GetFontFamily(string? fontFamliy) -> string!
```

Comment:
You need to keep this line in I think

Reply:
Hmm did I forget how this works again? I moved it to unshipped and added *REMOVED*? https://github.com/dotnet/maui/pull/28627/files#diff-9299a525ca3c3a4be75b890f814b2a67268bed18b12c531b805c40649b7eb447R3 

Is it supposed to stay in the Shipped but add *REMOVED*?

Reply:
yes it should stay on shipped, we never touch that file after is "ships"  and you just add to unshipped  with the *REMOVE*